packages/server/graphql/public/types/User.ts

```diff
@@ -196,7 +196,11 @@ const User: UserResolvers = {
   tier: ({tier, trialStartDate}) => {
     return getFeatureTier({tier, trialStartDate})
   },
-  billingTier: ({tier}) => tier
+  billingTier: ({tier}) => tier,
+  freeCustomRetroTemplatesRemaining: ({freeCustomRetroTemplatesRemaining}) =>
```

Comment:
+1 these aren't necessary if they exist on the source object, can remove (billingTier, too)